apps/docs-app/static/new/index.html

```diff
@@ -5,7 +5,7 @@
     <title>Analog - New Project</title>
     <meta
       http-equiv="refresh"
-      content="0; URL=https://stackblitz.com/github/analogjs/analog/tree/stackblitz/packages/create-analog/template-angular-v16?file=vite.config.ts&preset=node"
+      content="0; URL=https://analogjs.org/new"
```

Comment:
This link should remain the same, as its the page when you visit https://analogjs.org/new